src/const.ts

```diff
@@ -139,5 +139,11 @@ export const INPUT_PRECISION_SIZE = 6
 export const VALID_UNTIL_DEFAULT = '2880'
 export const VALID_FROM_DEFAULT = '30'
 export const WETH_ADDRESS_MAINNET = '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2'
+export const WETH_ADDRESS_RINKEBY = '0xc778417E063141139Fce010982780140Aa0cD5Ab'
```

Comment:
does this need to be toUpperCase or do we do this conversion elsewhere?

Reply:
doesn't matter, I think better to use the mixed case